--- a/bin/generate-dns-zones.php
+++ b/bin/generate-dns-zones.php
@@ -1,3 +1,4 @@
+#!/usr/bin/php
 <?php
 
 /*
@@ -7,8 +8,8 @@
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/LC/Node/autoload.php';
+$baseDir = '/usr/share/vpn-server-node';
 
 use LC\Common\Config;
 use LC\Common\HttpClient\CurlHttpClient;
--- a/bin/generate-firewall.php
+++ b/bin/generate-firewall.php
@@ -1,3 +1,4 @@
+#!/usr/bin/php
 <?php
 
 /*
--- a/bin/server-config.php
+++ b/bin/server-config.php
@@ -1,3 +1,4 @@
+#!/usr/bin/php
 <?php
 
 /*
@@ -7,8 +8,8 @@
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/LC/Node/autoload.php';
+$baseDir = '/usr/share/vpn-server-node';
 
 use LC\Common\Config;
 use LC\Common\HttpClient\CurlHttpClient;
--- a/libexec/client-connect.php
+++ b/libexec/client-connect.php
@@ -1,3 +1,4 @@
+#!/usr/bin/php
 <?php
 
 /*
@@ -7,8 +8,8 @@
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/LC/Node/autoload.php';
+$baseDir = '/usr/share/vpn-server-node';
 
 use LC\Common\Config;
 use LC\Common\HttpClient\CurlHttpClient;
--- a/libexec/client-disconnect.php
+++ b/libexec/client-disconnect.php
@@ -1,3 +1,4 @@
+#!/usr/bin/php
 <?php
 
 /*
@@ -7,8 +8,8 @@
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/LC/Node/autoload.php';
+$baseDir = '/usr/share/vpn-server-node';
 
 use LC\Common\Config;
 use LC\Common\HttpClient\CurlHttpClient;
