--- a/tests/ConnectionTest.php
+++ b/tests/ConnectionTest.php
@@ -19,7 +19,7 @@
     /** @var Connection */
     private $connection;
 
-    public function setUp()
+    public function setUp() : void
     {
         $this->connection = new Connection(
             new ServerClient(
--- a/tests/OpenVpnTest.php
+++ b/tests/OpenVpnTest.php
@@ -22,7 +22,7 @@
 
     private $tmpDir;
 
-    public function setUp()
+    public function setUp() : void
     {
         // create temporary directory
         $tmpDir = sprintf('%s/%s', sys_get_temp_dir(), bin2hex(random_bytes(16)));
