diff --git a/tests/ConnectionTest.php b/tests/ConnectionTest.php
index a7e507c..91b25e7 100644
--- a/tests/ConnectionTest.php
+++ b/tests/ConnectionTest.php
@@ -19,7 +19,7 @@ class ConnectionTest extends TestCase
     /** @var Connection */
     private $connection;
 
-    public function setUp()
+    public function setUp() : void
     {
         $this->connection = new Connection(
             new ServerClient(
diff --git a/tests/OpenVpnTest.php b/tests/OpenVpnTest.php
index d7d8e58..8293036 100644
--- a/tests/OpenVpnTest.php
+++ b/tests/OpenVpnTest.php
@@ -22,7 +22,7 @@ class OpenVpnTest extends TestCase
 
     private $tmpDir;
 
-    public function setUp()
+    public function setUp() : void
     {
         // create temporary directory
         $tmpDir = sprintf('%s/%s', sys_get_temp_dir(), bin2hex(random_bytes(16)));
